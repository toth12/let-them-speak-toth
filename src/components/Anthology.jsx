import React from 'react';
import Hero from './Hero';
import { Link } from 'react-router-dom';

const Anthology = props => (
  <div className='anthology'>
    <Hero text='Anthology of Fragments' />
    <div className='container'>
      <h2>A Word Tree Exploration of Holocaust Testimonies</h2>
      <p>Lorem ipsum dolor sit amet, arcu vulputate egestas dolor wisi, fugiat vestibulum etiam mattis sit, curabitur elit nulla vel, mus quis elit porttitor nulla. Amet elit in convallis, metus nam at sed, ipsum phasellus a dapibus, ornare in massa pharetra. Montes elementum pede integer ac, pulvinar aliquam non in augue, orci nulla eros nulla. Vulputate ut sed eget eu, dolor aliquam velit quisque a. Fermentum venenatis rhoncus vitae congue, suspendisse potenti pellentesque vestibulum.</p>
      <p>Amet a. Tempor eu. Dignissim nunc. Morbi faucibus, fermentum urna in, vitae nulla habitasse. A sit, vehicula eleifend, per massa vitae. Amet a. Tempor eu. Dignissim nunc. Morbi faucibus, fermentum urna in, vitae nulla habitasse.</p>
      <div className='center-text'>
        <Link className='button brown' to='/tree'>Let's explore</Link>
      </div>
    </div>
  </div>
)

export default Anthology;