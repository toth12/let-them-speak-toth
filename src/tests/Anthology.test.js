import React from 'react';
import Anthology from '../components/Anthology';
import { createBrowserHistory } from 'history'
import { Router } from 'react-router';
import renderer from 'react-test-renderer';

it('renders', () => {
  const tree = renderer
    .create(
      <Router history={createBrowserHistory()}>
        <Anthology/>
      </Router>
    ).toJSON();
  expect(tree).toMatchSnapshot();
});