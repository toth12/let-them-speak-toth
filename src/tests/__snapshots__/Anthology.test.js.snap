// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="anthology"
>
  <div
    className="hero background-image"
  >
    <h1>
      Anthology of Fragments
    </h1>
  </div>
  <div
    className="container"
  >
    <h2>
      A Word Tree Exploration of Holocaust Testimonies
    </h2>
    <p>
      Lorem ipsum dolor sit amet, arcu vulputate egestas dolor wisi, fugiat vestibulum etiam mattis sit, curabitur elit nulla vel, mus quis elit porttitor nulla. Amet elit in convallis, metus nam at sed, ipsum phasellus a dapibus, ornare in massa pharetra. Montes elementum pede integer ac, pulvinar aliquam non in augue, orci nulla eros nulla. Vulputate ut sed eget eu, dolor aliquam velit quisque a. Fermentum venenatis rhoncus vitae congue, suspendisse potenti pellentesque vestibulum.
    </p>
    <p>
      Amet a. Tempor eu. Dignissim nunc. Morbi faucibus, fermentum urna in, vitae nulla habitasse. A sit, vehicula eleifend, per massa vitae. Amet a. Tempor eu. Dignissim nunc. Morbi faucibus, fermentum urna in, vitae nulla habitasse.
    </p>
    <div
      className="center-text"
    >
      <a
        className="button brown"
        href="/tree"
        onClick={[Function]}
      >
        Let's explore
      </a>
    </div>
  </div>
</div>
`;
